pipeline.py

```diff
@@ -463,6 +484,21 @@ def pipeline(
         model_pvc_delete_task = DeletePVC(pvc_name=model_pvc_task.output)
         model_pvc_delete_task.after(final_eval_task)
 
+        generate_metrics_report_task = generate_metrics_report_op()
+        generate_metrics_report_task.after(output_mt_bench_task, final_eval_task)
```

Comment:
I don't think this requires to be run after `output_mt_bench_task`. That task only uploads artifacts. In fact both `output_mt_bench_task` and `generate_metrics_report_task` can run in parallel, they only read the same data.

Reply:
You are right. :) removing that requirement from `.after()`